gui/splash_hook.py

```diff
@@ -0,0 +1,64 @@
+import multiprocessing as mp
+import os
+import subprocess
+
+from mozregression import version
+
+
+class Splash:
+    # Milliseconds to pass before re-checking open files.
+    CHECK_MILLISECONDS = 500
+
+    def __init__(self):
+        # Must import here to avoid multiprocess forking issue with Core Foundation.
+        import tkinter
+
+        # Get parent PID which will be the one extracting files, etc...
+        self.ppid = str(os.getppid())
+
+        # Variable to keep track of last lsof output.
+        self.last_lsof = None
+
+        # Create the splash screen.
+        self.win = tkinter.Tk()
+        self.win.overrideredirect(True)
+        tkinter.Label(self.win, text=f"mozregression {version} starting, please wait...").pack(
+            pady=20, padx=20
+        )
+        self.win.eval("tk::PlaceWindow . center")
+        self.win.after(self.CHECK_MILLISECONDS, self.check_doneness)
+        self.win.mainloop()
+
+    def get_lsof(self):
+        """Return a list of files that are extracted by the bootloader."""
+        out = subprocess.check_output(["lsof", "-p", self.ppid])
+        lines = out.splitlines()
+
+        # _MEI is the prefix used in the temporary folder when extracting. See
+        # pyinstaller.org/en/stable/operating-mode.html#how-the-one-file-program-works.
+        relevant_lines = [line for line in lines if b"_MEI" in line]
```

Comment:
`_MEI` is not a prefix of the line itself, it is a prefix of a subdirectory within a directory hierarchy (depending on where the temporary folder ends up). This could be changed to something like `/_MEI` to be more specific, or, we could parse the line to extract the path, and then look for this part in the path explicitly.